Validation/RecoParticleFlow/plugins/PFClusterValidation.cc

```diff
@@ -0,0 +1,271 @@
+#include "Validation/RecoParticleFlow/plugins/PFClusterValidation.h"
+#include "DataFormats/HcalDetId/interface/HcalSubdetector.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include "DataFormats/HcalDetId/interface/HcalDetId.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "DataFormats/Common/interface/View.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+
+PFClusterValidation::PFClusterValidation(const edm::ParameterSet& conf) {
+  tok_evt_ = consumes<edm::HepMCProduct>(edm::InputTag("generatorSmeared"));
+
+  PFClusterECALTok_ =
+      consumes<reco::PFClusterCollection>(conf.getUntrackedParameter<edm::InputTag>("pflowClusterECAL"));
+  PFClusterHCALTok_ =
+      consumes<reco::PFClusterCollection>(conf.getUntrackedParameter<edm::InputTag>("pflowClusterHCAL"));
+  PFClusterHOTok_ = consumes<reco::PFClusterCollection>(conf.getUntrackedParameter<edm::InputTag>("pflowClusterHO"));
+  PFClusterHFTok_ = consumes<reco::PFClusterCollection>(
+      conf.getUntrackedParameter<edm::InputTag>("pflowClusterHF"));  // cms.InputTag("particleFlowClusterECAL");
+
+  outputFile_ = conf.getUntrackedParameter<std::string>("outputFile", "myfile.root");
+  mc_ = conf.getUntrackedParameter<std::string>("mc", "yes");
+
```

Comment:
@akanugan why do you use here a string instead of a bool?

Reply:
@jfernan2 Hi, I can make that change but the same string was used in another module which I carried forward. If it's necessary, I can change it.

Reply:
I think @akanugan meant 
https://cmssdt.cern.ch/lxr/source/Validation/CaloTowers/src/CaloTowersValidation.cc
Well, maybe we can fix this as well, i.e. use bool.
I think
@mariadalfonso
@abdoulline
@arapyan
will be happy if @akanugan does that?

Reply:
The change has been made and also it passed the tests.